tests/range_search_disk_index.cpp

```diff
@@ -226,21 +226,22 @@ int search_disk_index(diskann::Metric &metric, const std::string &index_path_pre
         auto mean_ios = diskann::get_mean_stats<uint32_t>(stats, query_num,
                                                           [](const diskann::QueryStats &stats) { return stats.n_ios; });
 
-        float mean_cpuus = diskann::get_mean_stats<float>(
+        double mean_cpuus = diskann::get_mean_stats<float>(
```

Comment:
why double?

Reply:
This was simply to align with the function signature of `get_mean_stats`, which returns a double. Alternatively, we can change the function signature to return a float. If there is a performance consideration in either case, I'm fine with it. Since this code does not impact the performance of building or searching the index, I'm inclined to leave it as double.